documentation/authentication.md

```diff
@@ -0,0 +1,135 @@
+Authentication
+================
+
+Authenticating requests to `dotnet-monitor` helps protect sensitive diagnostic artifacts from unauthorized users and lower privileged processes. `dotnet-monitor` can be configured to use either [Windows Authentication](#windows-authentication) or via an [API Token](#api-token-authentication). It is possible, although strongly not recommended, to [disable authentication](#disabling-authentication).
+
+## Windows Authentication
+We only recommend using Windows Authentication if you're running `dotnet-monitor` as a local development tool on Windows, for all other environments using an [API Token](#api-token-authentication) is recommended.
+
+Windows authentication doesn't require explicit configuration and is enabled automatically when running `dotnet-monitor` on Windows. When available, dotnet-monitor will authorize any user authenticated as the same user that started the `dotnet-monitor` process. It is not possible to disable Windows authentication.
```

Comment:
Not sure if it's mentioned, but we will not attempt windows auth if you run dotnet-monitor as administrator